src/main/java/gregicadditions/recipes/chain/Dyes.java

```diff
@@ -398,124 +402,158 @@ public static void init(){
                 .fluidInputs(Water.getFluid(1000))
+                .notConsumable(dust, RhodiumPlatedPalladium)
                 .outputs(SuccinicAcid.getItemStack(14))
                 .buildAndRegister();
 
-        // C4H6O4 + C3H8O -> C7H12O4 + H2O
+        // C4H6O4 + 2 C3H8O -> C10H18O4 + 2 H2O
         CHEMICAL_RECIPES.recipeBuilder().duration(260).EUt(1400)
                 .inputs(SuccinicAcid.getItemStack(14))
-                .fluidInputs(IsopropylAlcohol.getFluid(1000))
+                .fluidInputs(IsopropylAlcohol.getFluid(2000))
                 .fluidOutputs(Isopropylsuccinate.getFluid(1000))
-                .fluidOutputs(Water.getFluid(1000))
+                .fluidOutputs(Water.getFluid(2000))
                 .buildAndRegister();
 
+        //C7H8 + NH3 + 3 O -> C7H5N + 3 H2O
         LARGE_CHEMICAL_RECIPES.recipeBuilder().duration(200).EUt(1700)
                 .fluidInputs(Toluene.getFluid(1000))
                 .fluidInputs(Ammonia.getFluid(1000))
-                .fluidInputs(Oxygen.getFluid(1000))
+                .fluidInputs(Oxygen.getFluid(3000))
                 .fluidOutputs(Benzonitrile.getFluid(1000))
-                .fluidInputs(Water.getFluid(2000))
+                .fluidOutputs(Water.getFluid(3000))
                 .buildAndRegister();
 
+        //2 C7H5N + C10H18O4 -> 2 C3H8O + C16H12N2O2
         CHEMICAL_RECIPES.recipeBuilder().duration(250).EUt(1700)
                 .fluidInputs(Isopropylsuccinate.getFluid(1000))
                 .fluidInputs(Benzonitrile.getFluid(1000))
-                .input(dust,Sodium)
+                .notConsumable(SodiumEthoxide.getItemStack())
+                .notConsumable(IsopropylAlcohol.getFluid(10))
                 .fluidOutputs(IsopropylAlcohol.getFluid(1000))
                 .outputs(Diketopyrrolopyrrole.getItemStack())
                 .buildAndRegister();
 
+        //2 CH2O + 2 HCN + 2 C6H7N + 2O -> C16H10N2O2 + 2 H2O + 2 NH3
         LARGE_CHEMICAL_RECIPES.recipeBuilder().duration(240).EUt(1100)
-                .fluidInputs(Formaldehyde.getFluid(1000))
-                .fluidInputs(HydrogenCyanide.getFluid(1000))
+                .fluidInputs(Formaldehyde.getFluid(2000))
+                .fluidInputs(HydrogenCyanide.getFluid(2000))
                 .fluidInputs(Aniline.getFluid(1000))
-                .fluidInputs(PotassiumHydroxide.getFluid(1000))
+                .fluidInputs(Oxygen.getFluid(2000))
+                .notConsumable(PotassiumHydroxide.getFluid(10))
                 .notConsumable(SodiumAzanide.getItemStack())
                 .notConsumable(dust, SodiumHydroxide)
-                .fluidInputs(HydrochloricAcid.getFluid(1000))
-                .outputs(Indigo.getItemStack(2))
-                .outputs(OreDictUnifier.get(dust, RockSalt))
-                .fluidOutputs(Water.getFluid(1000))
-                .fluidOutputs(Ammonia.getFluid(1000))
+                .outputs(Indigo.getItemStack(15))
```

Comment:
This one could easily just be 30, without having to divide all by two
```suggestion
                .outputs(Indigo.getItemStack(30))
```